js/src/components/RelatedKeyphraseModalContent.js

```diff
@@ -43,11 +45,74 @@ RelatedKeyphraseModalContent.propTypes = {
 	keyphrase: PropTypes.string,
 	relatedKeyphrases: PropTypes.array,
 	renderAction: PropTypes.func,
+	data: PropTypes.object,
 };
 
 RelatedKeyphraseModalContent.defaultProps = {
 	isLoading: true,
 	keyphrase: "",
 	relatedKeyphrases: [],
 	renderAction: null,
+	data: {
+		data: {
+			columnNames: [
+				"Keyword",
+				"Search Volume",
+				"Trends",
+			],
+			rows: [
+				[
+					"and you and you and you and you",
```

Comment:
I don't think we want those default rows 😅

Reply:
I think we can replace the default rows in the issue that adds the request data to the table :) ([P3-7](https://yoast.atlassian.net/secure/RapidBoard.jspa?rapidView=61&projectKey=P3&modal=detail&selectedIssue=P3-7)). I've updated the issue.

Reply:
Yeah, of course these defaults are only for testing purposes.